src/canvasPanelComponents/slideshow/src/components/CanvasNavigation/CanvasNavigation.js

```diff
@@ -23,6 +47,7 @@ class CanvasNavigation extends Component {
           onClick={ev => {
             ev.preventDefault();
             previousRange();
+            this.goToSlide(currentIndex - 1);
```

Comment:
I think the `currentIndex` might be changed from outside of this component, so there's no guarantee that the slide will always be updated. We could use `componentWillReceiveProps` (or the modern eq. I forget), and set it when the `currentIndex ` prop changes.